config/logstash.yml

```diff
@@ -85,6 +85,11 @@
 #
 # config.debug: false
 #
+# When enabled, process escaped characters such as \n and \" in strings in the
+# pipeline configuration files.
+#
+# config.strings.support_escapes: false
```

Comment:
`strings` is in reference to string syntax in the grammar, not the "config string" (aka `-e`).

This is perhaps a bad name.

I wanted to imply that this was only for String grammar (https://github.com/elastic/logstash/blob/master/logstash-core/lib/logstash/config/grammar.treetop#L75-L85) and nowhere else.

I'm open to calling it `config.support_escapes`